frontend/src/features/videos/VideoCard.tsx

```diff
@@ -223,6 +245,14 @@ function VideoCard({
 
   const nbRatings = video.rating_n_ratings;
   const nbContributors = video.rating_n_contributors;
+  if (nbContributors != null && nbContributors <= 1) {
+    unsafe = true;
+    unsafe_cause = t('video.unsafeNotEnoughContributor');
+    if (total_score < 0) {
+      unsafe = true;
+      unsafe_cause = t('video.unsafeNegativeRating');
+    }
```

Comment:
This condition should not be nested: a video with a negative score and 2 contributors should be considered "unsafe" too.

Reply:
Done too